src/components/Survey.js

```diff
@@ -1,65 +1,110 @@
-import { NONE } from 'apisauce';
+
 import React, { useState } from 'react'
-import { SafeAreaView, StyleSheet, ScrollView, View, Text, Button, FlatList, TouchableOpacity, TextInput,Alert } from 'react-native';
-import {Field, reduxForm } from 'redux-form'
+import { SafeAreaView, StyleSheet, ScrollView, View, Text,TouchableOpacity ,Alert } from 'react-native';
+import {Field, reduxForm, destroy } from 'redux-form'
+import { connect, useDispatch } from 'react-redux';
+import {saveUser}  from '../Redux/Actions/operationActions'
+import { useNavigation } from '@react-navigation/native';
+import axios from 'axios'
+import RenderField from '../components/RenderField'
+import {required, checkTel, checkUser, telLength ,checkFields} from '../components/Validations'
 
 
-const renderField = ({label}) => {
-const [flag, setFlag] = useState(false)
 
-  return (
-    <>
-      <View style={{flexDirection:'row',height:50, alignItems:'center'}}>
-        <View>
-        <Text style={{fontSize:14, fontWeight:'bold', width:80}}>{label}:</Text>
-        </View>
-        
-        <View>
-        <TextInput style={label === 'Comentario' ? styles.textArea : styles.input } multiline={label === 'Comentario' ? true : false}  numberOfLines={10}/>
-        </View>
-      </View>
-    </>
-  )
-}
+let Survey  = (props) => {
+  
+  const [userName ,setUserName] = useState('')
+  const [cell ,setCell] = useState('')
+  const [comment ,setComment] = useState('')
 
 
-let Survey = props => {
-/* 
-const {handleSubmit} = props */
+  const {handleSubmit} = props 
+  const navigation = useNavigation();
+  const dispatch = useDispatch() 
+ 
+  
 
-const handleSubmit = (valueP) => {
-  Alert.alert("Mis valores props handleSubmit son ",valueP)
+  const submit = async (values) => {
+    
+    if(!checkFields(userName,cell,comment)){
+      Alert.alert("Tiene que llenar todos los campos")
+      return 0
+    
+    }else{
+
+    const keyResponse = { input_values: { values } };
+    const response = await axios.post('https://private-e75208-formresponse.apiary-mock.com/form_response', keyResponse);
+    if(response != null){
+      Alert.alert("¡Gracias por el mensaje!")
+    }
+  }
 }
 
+  const handleOnChange = (user) => {
+    setUserName(user)
+  }
+
+  const savePhone = (user) => {
+    setCell(user)
+  }
+  
+  const saveComment = (user) => {
+    setComment(user)
+  }
```

Comment:
This is not necessary, redux-forms already saves input and your component receives it as `input: {value}`,